library/neo4j.py

```diff
@@ -207,13 +207,17 @@ def add_to_db(self, person_list, events_list, attendance_list):
     @staticmethod
     def collate_schedule_response(records):
         def key(record):
-            return record['person.name'] + "||" + record['person.email'] + "||" + record['event.name'] + "||" + str(record['event.start']) + "||" + str(record['event.end'])
+            start = record['event.start'].strftime("%s")
```

Comment:
This old format would have been timezone sensitive and failed to collate properly if entries were from people with different iso strings